### PR TITLE
Refactor config initialization

### PR DESCRIPTION
**Pull request by louyuting**
### Special notes for reviews

The wiki: [https://github.com/alibaba/sentinel-golang/wiki/通用配置项](https://github.com/alibaba/sentinel-golang/wiki/通用配置项)
had been updated.

### CODE REVIEW

**Comment by louyuting** [comment]
The travis seems to have stopped working.
@sczyh30

File: core/config/entity.go [hunk 1]
```diff
@@ -19,6 +19,8 @@ type SentinelConfig struct {
 		// Type indicates the classification of the service (e.g. web service, API gateway).
 		Type int32
 	}
+	// EnableMetricLog represents whether to enable metric monitor log
+	EnableMetricLog bool `yaml:"enableMetricLog"`
```
Comment (sczyh30) [inline core/config/entity.go:23]: Maybe it's not needed. Setting `sentinel.log.metric.flushIntervalSec` to 0 is enough.

Comment (louyuting) [reply core/config/entity.go:23]: Make sense.
I will remove this config field.

**Comment by sczyh30** [comment]
Thanks!

**Pull request merged**